src/components/pages/AddChild/RenderAddChild.js

```diff
@@ -11,7 +11,7 @@ const { Title } = Typography;
 
 const RenderAddChild = props => {
   return (
-    <Layout className="add-child">
+    <Layout className="edit-players">
```

Comment:
Nice work with the classname change for the refactor

Reply:
I take it the plan is to just consider all additions and edits to be "edit"s and that is the reason for the class name change? I realize it is just a CSS class name but it does catch the eye in an Add component.